src/Calculator.jsx

```diff
@@ -1,5 +1,7 @@
 import React, { Component } from 'react';
 import './Calculator.css';
+import { FaWeightHanging, FaWeight } from "react-icons/fa";
+import { IoFastFood } from "react-icons/Io";
```

Comment:
You need to fix the name of the module because the `IoFastFood` icon is located in the `react-icons/io5` module which is described here: https://react-icons.github.io/react-icons/icons?name=io5
```suggestion
import { IoFastFood } from "react-icons/io5";
```